refocus-bdk-server.js

```diff
@@ -362,7 +374,7 @@ module.exports = (config) => {
     });
 
     socket.on('connect', () => {
-      socket.emit('auth', token);
+      socket.emit('auth', BOT_INSTALL_TOKEN);
     }).on('authenticated', () => {
       logger.info('Socket Connected');
     }).on('auth error', (err) =>
```

Comment:
Should also wrap these handlers in the toggle

Reply:
Done 👍🏻